src/css/application.css

```diff
@@ -623,6 +623,11 @@ body {
 .console__row {
   padding-left: 1em;
   position: relative;
+  display: inline-flex !important;
+}
+
+.console__row > li {
```

Comment:
Just to make sure, the `<li>` in question is coming from the `<Inspector>`, right?